### DISCUSSION TITLE
Library issues

### DISCUSSION THREAD

**Discussion by SZLNET** [rern/rAudio#400] [General]
Hi, 
I am new here. I started a journey several month ago to find the best digital sound on the internet for my music moments.
I arrived here via RueAudio, it was difficult to understand thios develeopment from there. But I started to test for myself and I can tell you that this app is of fabulous sound. I am not a developer, but an advanced user, probably good for innocent tests.
My experiences so far: rAudio 1 so far is an excellent sound solution, what concerns the engine part under the hood. Do not know how this happens, but the professional skills are very visible / hearable. I use the following setup: Raspberry Pi 4 +USB stick as device with a pro audio interface (Rode AI-1) as USB DAC + Beyerdynamic DT 880 as headphones, great tools for music. All  I think that rAudio 1 is a decent partner to these. My only reserve so far is ergonomy, especially around the Library Albums part. I see several points of improvements here. E.g. coverart thumbnails refresh button distributed the coverart of the first album to nearly all !! of my 289 albums which had already coverart, How to restore or delete them? Also, why not the right cobcertart? Many international albuls like Guitars from Mike Oldfiled etc. Selecting multiple tracks for playlist is also unclear or I miss something, but ergonomy lacks. Do not misunderstand, I am hugely impressed by the sound engine, it is better than anything I tried recently, including all those know names. A more streamline GUI would make this a perfect solution for guys like me. If you maybe like to hear about the testing experiences of a real user, I can send you my observations later, too, to give feedback. I know the product is in development yet and certain parts are yet "under improvement". And maybe I should not be here...  But rAudio is already my top choice for listening to music. Please comment at your convenience. Laszlo

**Comment by AndyFx12** [comment]
Hi SZLNET,

your reported coverart-problem is not reproducable on my system (830 albums with ~15000 files). JPG´s will be created at corresponding album-music-folder. Existing picture files shouldn´t be touched.
Please post more details.

I´m impressed of rAudio´s GUI. You can add single tracks/whole albums/saved playlist to your current playlist. Once you have created your personal playlist, you can save it for using later again.

regards, Andy

**Comment by SZLNET** [comment]
Hi AndyFx12,

Thanks for your comments. In fact, based on your message, I realized one imprtant thing. My test library of 3000 songs is within one folder on a USB stick attached directly. File names deliver all identification info, filtering etc. 
So probably this is why getting coverart for the folder supplied the same coverart for all songs as they are all in one folder. That is a possible configuration generally. However I have to reorganize my collection into folders, I suppose it will not produce the same problem. I will come back with the results, but this must work out.
regards, laszlo

**Comment by SZLNET** [comment]
Hi, 
While testing, I come across a repeating problem, this is Album selector. It is to be noticed that all my files in one folder, tagged quite precisely.  So when I browse by AlbumArtist, the selected album of that artist is shown correctly. But if I want to select the same album in Album mode, it gives back all the time the whole content of my single folder... all those several thousand songs, so not filtered. Also, Album mode does not show any coverart, while they are there in files and also visible in AlbumArtist mode. Album covers in Album mode is normally my preferred selecting method, but it does not work yet here. Definitely, Album mode has some filteirng issue. Or do I miss something? Could you have a look? thnx & regards, laszlo
